server/migrations/20200708183925-subscription-model-upgrade.js

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable quotes */
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, DataTypes) => {
```

Comment:
I would recommend doing this all as part of a single transaction, so it rolls back in case of error.

Reply:
👍 done!